metricsmentor/main/migrations/0002_answer.py

```diff
@@ -1,28 +0,0 @@
-# Generated by Django 4.2.11 on 2024-05-28 18:18
```

Comment:
I'm not sure the context here, but deleting migration files is generally bad practice unless you have a very specific reason to do so. If this was deployed on staging/production, these changes were already made to the database, and removing this file could cause things to get out of sync and cause confusing issues like I saw on carr. So it would be best to make a new migration file removing these fields if that's the intent here, and keep things always moving forward.

Or, if you plan on just handling this all manually, and wiping the database yourself to start fresh without this migration - that could be fine too, I just wanted to make sure you know what you're doing.

Reply:
The latter option of just wiping the database def makes sense at the beginning of a project here - sometimes your data model changes unexpectedly and it's nice to just restructure things on a clean slate like this, so that's fine.

Reply:
Yeah I already truncated a table manually. There were some annoying things happening with the foreign keys and we decided to just start from scratch here since there's not a lot of data saved anyway.